pyemma/coordinates/data/_base/iterable.py

```diff
@@ -351,7 +351,7 @@ def _n_chunks(self):
         return min(n1, n2)
 
     def __len__(self):
-        return min(self._it.trajectory_lengths(), self._it_lagged.trajectory_lengths())
+        return int(min(self._it.trajectory_lengths(), self._it_lagged.trajectory_lengths()))
```

Comment:
maybe 2.7 requires `np.min(arr)` instead of `arr.min()`? Seemed to pass travis before